curriculum/challenges/english/08-coding-interview-prep/algorithms/implement-bubble-sort.english.md

```diff
@@ -43,14 +43,10 @@ tests:
 
 ```js
 function bubbleSort(array) {
-  // change code below this line
-
-  // change code above this line
```

Comment:
Please do not remove `// change code above this line` from the challenge seed code.